Makefile

```diff
@@ -274,7 +274,10 @@ secrets:
 	hack/secrets.sh $(kerberos_id) $(kubeconfig_path) $(bw_password_path)
 .PHONY: secrets
 
+serviceaccount-secret-rotation:
+	go get github.com/openshift/ci-tools/cmd/serviceaccount-secret-rotation-trigger
+	time serviceaccount-secret-rotation-trigger --dry-run=false  --namespace=ci 2>&1|tee /tmp/out.json
```

Comment:
Just curious, why the `time` and the `tee`? Debugging leftovers?

Reply:
Yeah, updated the PR to instead just create a periodic